<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDetailsModal</class>
 <widget class="QDialog" name="ClientDetailsModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>254</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="clientVerticalLayout1">
    <item>
     <widget class="QLabel" name="clientNameLabel">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="clientNameEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="clientAgeLabel">
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="clientAgeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="clientPhoneNumberLabel">
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="clientPhoneNumberEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="clientEmailLabel">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="clientEmailEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
